Sources/Danger/BitBucketCloud.swift

```diff
@@ -71,7 +71,7 @@ extension BitBucketCloud {
         public let destination: MergeRef
 
         /// PR's ID
-        public let id: Int // swiftlint:disable:this identifier_name
+        public let id: Int 
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/BitBucketCloud.swift;
  Line: 74;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)